packages/fxa-settings/src/components/Settings/AccountRecoveryWizard/index.tsx

```diff
@@ -0,0 +1,133 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, { useState } from 'react';
+import { RouteComponentProps } from '@reach/router';
+import { FlowContainer } from '../FlowContainer';
+import { ProgressBar } from '../ProgressBar';
+
+export const AccountRecoveryWizard = (props: RouteComponentProps) => {
+  const [currentStep, setCurrentStep] = useState(1);
+  /*
+    Making this a component (as opposed to writing it directly into the PageRecoveryAdd view) will hopefully allow us to separate the logic a little more from the UI.
```

Comment:
Will this Wizard be reusable for both Adding and Changing recovery keys? Would the logic be passed down from the page level to the components with the wizard acting as middleman?

Reply:
No -- this is really just an incapsulation of the UI aspect of the RecoveryKeyAdd page, so that logic can be separated out into the parent component. I think it's more trouble than it's worth to make a separate wizard component that we reuse across those wizards since it's pretty much just a switch case and a single piece of state, whereas making an abstract wizard wrapper can actually be a bit tricky. But I could be missing something!

Reply:
I guess what I'm wondering is if the code here should just directly be in `PageRecoveryKeyCreate` (or other name), if `AccountRecoveryWizard` is not reusable? 
Or am I misunderstanding, and this is a wrapper around individual pages (i.e., each step is a Page) and we won't have a `PageRecoveryKeyCreate` at all?